src/app/view/about/about-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
+import { Routes, RouterModule } from '@angular/router';
+import {AboutComponent} from "./about.component";
```

Comment:
Linting error here, please add spacing inside the brackets: `{ AboutComponent }`

Reply:
Also use single quotes `'`